docs/1-simple-conversation-objc.md

```diff
@@ -0,0 +1,282 @@
+
+### 2.1 Start a new project
+
+Open Xcode and start a new project. We'll name it "simple-conversation-objc".  
+
+## Interoperability with Objective-C
+
+While the Swift iOS SDK is written in Swift, there is a support for interoperability in Objective-C. 
+
+### Making the Swift Classes available to Objective-C Files
+
+The process of setting up interoperability in Objective-C is identical for nearly all environments. 
+
+1. Open up the Objective-C project. 
+2. Add a new Swift file to the project. In the menu select File>New>File…. Select Swift File instead of a Cocoa Touch File. 
+3. Name the file `Stitch-Swift`.
+4. A dialogue box will appear in Xcode so please make sure to select "Create Bridging Header".
+5. Go to your project's "Build Settings" and switch to "All" instead of "Basic", which is the default option. Under "Packaging" turn on  "Defines Module" by chaning "No" to "Yes". Changing this parameter enables us to use Swift classes inside of Objective-C files.
+6. Inside of "Build Settings" make sure to look for "Product Module Name" in the "Packaging" section so that you can copy the "Product Module Name" exactly. 
+
+### 2.2 Adding the Nexmo iOS Conversation SDK to Cocoapods
+
+Navigate to the project's root directory in the Terminal. Run: `pod init`. Open the file entitled `PodFile`. Configure its specifications accordingly: 
+
+```bash
+# Uncomment the next line to define a global platform for your project
+platform :ios, '10.0'
+
+source "https://github.com/Nexmo/PodSpec.git"
+source 'git@github.com:CocoaPods/Specs.git'
+
+target 'simple-conversation-objc' do
+  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+  use_frameworks!
+  pod "Nexmo-Stitch", :git => "https://github.com/nexmo/conversation-ios-sdk.git", :branch => "master" # development
+end
+```
+
+## Accessing a Swift Class in Objective-C
+
+To start programming in Objective-C with the Stitch Swift iOS SDk, add a CocoaTouch class called `LoginVC` as a subclass of UIViewController. Program the interface file in the following way:
+
+1. Import the Stitch iOS SDK in the interface file.
+2. In the interface file's interface add a property for the client. 
+
+Your code should look like this: 
+
+```Objective-C
+@import Stitch;
+
+@interface LoginVC : UIViewController
+
+@property (retain, nonatomic) NXMConversationClient *client;
+
+@end
+```
+
+Program the implementation file in the following way: 
+
+1. Import the LoginVC interface file. 
+2. Synthesize the client. 
+3. Initialize the client after the controller's view is loaded into memory. 
+
+Your code should look like this: 
+
+
+```Objective-C
+#import "LoginVC.h"
+
+@interface LoginVC ()
+
+@end
+
+@implementation LoginVC
+
+@synthesize client;
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    
+    client = [NXMConversationClient instance];
+}
+
+@end
+```
+
+### 2.4 Creating the login layout
```

Comment:
seems there is some sort of parsing error after the first code sample, can you look into that?

Reply:
I didn't see in MacDown.